### ISSUE TITLE
Updates to 2.0.0-beta-alpine?

### ISSUE THREAD

**Issue by qm3ster** [influxdata/influxdb#17280]
Henlo, I noticed that `nightly-alpine` builds just as often and just as well as `nightly`, but `2.0.0-beta-alpine` has not been updated for months (unlike the debian-based image).
Is this intentional?

**Comment by russorat** [comment]
@qm3ster thanks for opening this! we currently don't build an alpine version of 2.0.0-beta, but we are planning to before our GA release.

**Comment by rawkode** [comment]
We need to remove `goreleaser` from the code, which is in progress:

https://github.com/influxdata/influxdb/pull/18177

Once this is merged, I can push the Alpine changes back in

**Comment by russorat** [comment]
Much has changed since the beta for this, so i'm closing for now. Please open a new issue if this is still a problem.
